### PR TITLE
Fix code style

### PR DESCRIPTION
**Pull request by revolter**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -14,12 +14,12 @@ let package = Package(
     ],
     targets: [
         .target(name: "App", dependencies: ["Vapor", "LeafProvider"],
-               exclude: [
-                   "Config",
-                   "Database",
-                   "Public",
-                   "Resources"
-               ]),
+                exclude: [
```
Comment (tanner0101) [inline Package.swift]: this should actually be tabbed over to the left according to vapor style (which we need to document somewhere)

like this:

```swift
// swift-tools-version:4.0
import PackageDescription

let package = Package(
    name: "VaporApp",
    products: [
        .library(name: "App", targets: ["App"]),
        .executable(name: "Run", targets: ["Run"])
    ],
    dependencies: [
        .package(url: "https://github.com/vapor/vapor.git", .upToNextMajor(from: "2.2.0")),
        .package(url: "https://github.com/vapor/leaf-provider.git", .upToNextMajor(from: "1.1.0")),
    ],
    targets: [
        .target(
            name: "App", 
            dependencies: ["Vapor", "LeafProvider"],
            exclude: ["Config", "Database", "Public", "Resources"]
        ),
        .target(name: "Run", dependencies: ["App"]),
        .testTarget(name: "AppTests", dependencies: ["App", "Testing"])
    ]
)
```

Comment (revolter) [reply Package.swift]: Then https://github.com/vapor/api-template/blob/d07ad4fda81726a0f21da5403c82fc6944276768/Package.swift#L16-L21 needs to be fixed too, right?

Comment (tanner0101) [reply Package.swift]: yes

Comment (revolter) [reply Package.swift]: Will update this PR and create a related one there soon.

**Comment by revolter** [comment]
@tanner0101, Done

**Comment by tanner0101** [comment]
thanks!

**Pull request merged**